### ISSUE TITLE
v.14.4 PopoverMenu with default props won't open on focus

### ISSUE THREAD

**Issue by bdares** [instructure/instructure-ui#2]
Related to https://github.com/instructure/instructure-ui/issues/1

Can repro directly on demo site [here](http://instructure.github.io/instructure-ui/#PopoverMenu) - close the menu and attempt to re-open it.

When I render:

```
<PopoverMenu label={ <span>Foo</span> }>
  <MenuItem><Link onClick={console.log}>MENU</Link></MenuItem>
</PopoverMenu>
```

I end up with a menu button as expected. Focusing the button via keyboard controls does not display the menu.

Apparent cause is that `OverlayTrigger` listens for [`onBlur`](https://github.com/instructure/instructure-ui/blob/v0.14.4/lib/util/OverlayTrigger.js#L230) and closes the menu when the menu _button_ is blurred.  However, opening the menu will trigger `Menu`'s [`focusIfNeeded`](https://github.com/instructure/instructure-ui/blob/v0.14.4/lib/components/Menu/index.js#L112), causing focus to be placed on the menu item, which blurs the menu trigger, causing the menu to close.

**Comment by junyper** [comment]
@bdares We're currently reworking the Popover based components PopoverMenu and Tooltip. We'll make sure we fix these bugs. You should see updates sometime in the next couple of weeks. Thanks for opening this issue!

**Comment by junyper** [comment]
I think this is fixed in v0.15.0
